src/main/java/com/mraof/minestuck/alchemy/CardCaptchas.java

```diff
@@ -49,8 +47,28 @@ public static String getCaptcha(Item item, MinecraftServer server)
 		MSExtraData data = MSExtraData.get(server);
 		CardCaptchas captchas = data.getCardCaptchas();
 		
-		if(captchas.captchasMap.containsKey(item))
-			return captchas.captchasMap.get(item);
+		if(PredeterminedCardCaptchas.GetData().containsKey(item))
+		{
+			if(captchas.captchasMap.containsKey(item))
+			{
+				captchas.captchasMap.remove(item);
+				data.setDirty();
```

Comment:
I had the thought that this should probably be logged as a warning. Same thing for the corresponding conflict removal in `getItemFromCaptcha()`.